pkg/controller/storage/mem/storage_binding.go

```diff
@@ -38,35 +38,62 @@ func newMemStorageBinding() *memStorageBinding {
 }
 
 func (m *memStorageBinding) List() ([]*servicecatalog.Binding, error) {
-	ret := make([]*servicecatalog.Binding, len(m.bindings))
+	copy := make([]*servicecatalog.Binding, len(m.bindings))
 	i := 0
-	for _, val := range m.bindings {
-		ret[i] = val
+	for _, bin := range m.bindings {
```

Comment:
No. Because `m.bindings` is a map. If we did as you suggested, `i` would range over string keys instead of numeric indices.